srcpkgs/qtile/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'qtile'
+pkgname=qtile
+version=0.21.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools_scm python3-cairocffi python3-xcffib python3-wheel pkg-config"
+makedepends="python3-devel libffi-devel pulseaudio-devel"
+depends="python3-cairocffi python3-xcffib pango gdk-pixbuf"
+short_desc="Full-featured tiling window manager written and configured in Python"
+maintainer="Kai Stian Olstad <kai.stian.olstad@gmail.com>"
+license="MIT"
+homepage="http://www.qtile.org/"
+changelog="https://raw.githubusercontent.com/qtile/qtile/v${version}/CHANGELOG"
+distfiles="${PYPI_SITE}/q/qtile/qtile-${version}.tar.gz"
+checksum=93e1f8216c669b5570333607d5d06de7d7f16033b0e73cf34c896459f2df5254
+
+# Pulseaudio is only needed for an optional widget
+skiprdeps="/usr/lib/python3.10/site-packages/libqtile/widget/_pulse_audio.abi3.so"
```

Comment:
Not sure why you don't want to depends on pulseaudio, but please don't.
Either split it into subpackage or make `qtile` depends on libpulseaudio, it's not that big.

Reply:
Only reason was that the widget is optional, for me personally I use it and have libpulseaudio installed.
I can just remove the line and xbps-src automatically add libpulseaudio as runtime dependency on the package.

Reply:
Remove it then, whoever want to complain, you can point them to me.